wled00/data/index.js

```diff
@@ -1694,15 +1689,15 @@ function makeSeg()
 	gId('segutil').innerHTML = cn;
 }
 
-function resetUtil()
+function resetUtil(off=false)
 {
-//	gId('segutil').innerHTML = '<button class="btn btn-s" onclick="makeSeg()"><i class="icons btn-icon">&#xe18a;</i>segment</button>';
-	gId('segutil').innerHTML = '<div class="seg btn btn-s" style="border-radius:24px;padding:0;">'
+	gId('segutil').innerHTML = `<div class="seg btn btn-s ${off?'off':''}" style="border-radius:24px;padding:0;">`
 	+ '<label class="check schkl"><input type="checkbox" id="selall" onchange="selSegAll(this)"><span class="checkmark"></span></label>'
-	+ '<div class="segname" onclick="makeSeg()"><i class="icons btn-icon">&#xe18a;</i>Add segment</div></div>';
+	+ `<div class="segname" ${off?'':'onclick="makeSeg()"'}><i class="icons btn-icon">&#xe18a;</i>Add segment</div></div>`;
```

Comment:
Is is correct to use a "back-tick" here? Just curious, as I'm not so much experienced with JS.

Reply:
Back-tick (as I learned during my WLED tenure) is a special string marker. Anything within back-ticks is paresed in a similar way as PHP parses strtings. `${...}` is the parsing trick. Within it, regular JS can be embedded.